README.md

```diff
@@ -23,14 +23,141 @@ Or install it yourself as:
 
     $ gem install bitreserve
 
-## Usage
+# Contents
 
-TODO: Write usage instructions here
+* [Usage](#usage)
+* [Authentication](#authentication)
+  * [OAuth2](#oauth2)
+  * [Basic Auth](#basic-auth)
+  * [Personal Access Token](#personal-access-token)
+    * [Via argument](#via-argument)
+    * [Via environment variable](#via-environment-variable)
+* [API calls](#endpoints)
+  * [Currencies](#currencies)
+  * [Tickers](#tickers)
+  * [Entities](#entities)
+  * [Cards](#cards)
+  * [Transactions](#transactions)
+  * [Contacts](#contacts)
+  * [Users](#users)
+  * [Transparency](#transparency)
+* [Testing](#testing)
+* [Contributing](#contributing)
 
-## Contributing
+# Usage
 
-1. Fork it ( https://github.com/[my-github-username]/bitreserve/fork )
+To use the gem, you have to instantiate a client. All API calls are made from there. Here's a minimal working example:
+
+```ruby
+require 'bitreserve'
+
+client = Bitreserve::Client.new
+puts client.all_tickers
+```
+
+All subsequent API calls are made from the client instance. For example:
+
+```ruby
+client.me
```

Comment:
This should probably be removed from here, as it requires authentication, as I mentioned earlier.